pkg/controller/datavolume/controller-base.go

```diff
@@ -300,6 +310,30 @@ func addDataVolumeControllerCommonWatches(mgr manager.Manager, dataVolumeControl
 		return err
 	}
 
+	// Watch for PV updates to reconcile the DVs waiting for available PV
+	if err := dataVolumeController.Watch(&source.Kind{Type: &corev1.PersistentVolume{}}, handler.EnqueueRequestsFromMapFunc(
+		func(obj client.Object) (reqs []reconcile.Request) {
+			pv := obj.(*corev1.PersistentVolume)
+			dvList := &cdiv1.DataVolumeList{}
+			if err := mgr.GetClient().List(context.TODO(), dvList, client.MatchingFields{dvPhaseField: ""}); err != nil {
+				return
+			}
+			for _, dv := range dvList.Items {
+				storage := dv.Spec.Storage
```

Comment:
It's not needed as we `create the pvc even if the storage class doesn't exist` if `AccessModes` is set (otherwise k8s PVC validation fails). `PVC/DV` phase will be `Pending` until a satisfying `SC/PV` is found.